test/classes.coffee

```diff
@@ -464,7 +466,7 @@ test "ensure that constructors invoked with splats return a new object", ->
   # Ensure that constructors invoked with splats cache the function.
   called = 0
   get = -> if called++ then false else class Type
-  new get() args...
+  new (get()) args...
```

Comment:
`()` was necessary to ensure the `new` is called on the result of `get`, otherwise you get a "constructors must be called with new" error.